server/stycoon/game.go

```diff
@@ -165,7 +170,14 @@ func (game *Game) GetShips(playerId *int64, resources *map[int]map[string]*Tradi
 
 func (game *Game) getPlayerCommands(playerId int64) (map[int]Command, error) {
 	commands := make(map[int]Command)
-	rows, err := game.db.Query("select object.`id`, `type`, `target`, `resource`, `amount`, `class` from t_command join t_object object on t_command.ship = object.id where object.`owner` = ?", playerId)
+	var rows *sql.Rows
+	var err error
+	// return all commands for history report
+	if playerId == -1 {
+		rows, err = game.db.Query("select object.`id`, `type`, `target`, `resource`, `amount`, `class` from t_command join t_object object on t_command.ship = object.id")
```

Comment:
put common part of the query in a constant/variable

Reply:
done - I agree with it OTOH syntax highlighting does not work for the string concat now :/